mittab/apps/tab/views.py

```diff
@@ -248,4 +272,47 @@ def view_scratches(request):
                              {'item_type':'team',
                               'title': "Viewing All Scratches for Teams",
                               'item_list':c_scratches}, context_instance=RequestContext(request))
+
+def upload_data(request):
+    if request.method == 'POST':
+      form = UploadDataForm(request.POST, request.FILES)
+      if form.is_valid():
+        judgeErrors = scratchErrors = teamErrors = []
+        importName = ''
+        results = ''
+        #TODO: Could probably turn these checks into a loop
+        if 'teamFile' in request.FILES:
```

Comment:
So I'm a little confused how we get schools. I don't think I see where that automatically gets created from team/judge schools, but I might just be missing it.